src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -200,6 +200,63 @@ public static void UpdateReturnType(this AppCompatEditText editText, IEntry entr
 			editText.ImeOptions = entry.ReturnType.ToNative();
 		}
 
+		[PortHandler]
+		public static void UpdateCursorPosition(this AppCompatEditText editText, IEntry entry)
+			=> UpdateCursorSelection(editText, entry);
+
+		[PortHandler]
+		public static void UpdateSelectionLength(this AppCompatEditText editText, IEntry entry)
+			=> UpdateCursorSelection(editText, entry);
+
+		/// <summary>
+		/// Updates both the <see cref="IEntry.CursorPosition"/> & <see cref="IEntry.SelectionLength"/> properties.
```

Comment:
i think this isn't building because of a issue in this comment 

```
  D:\a\1\s\src\Core\src\Platform\Android\EditTextExtensions.cs(212,60): error CS1570: XML comment has badly formed XML -- '{0}' [D:\a\1\s\src\Core\src\Core-net6.csproj]
``` 
https://dev.azure.com/xamarin/public/_build/results?buildId=39696&view=logs&j=9cf5f8f4-c6e1-5b84-8b95-5f96f5b2da57&t=8829d08b-fb17-549e-9e41-6c6e0698f560&l=9395

Reply:
That XML comment issue should be fixed for both Android and iOS now.